File: Server/src/WeaponsManager.ts
import { AbstractModManager } from "./AbstractModManager"
import { Constants } from "./Constants"
import { Helper } from "./Helper"


export class WeaponsManager extends AbstractModManager
{
    protected configName: string = "WeaponsConfig"

    protected afterPostDB(): void
    {
        Helper.iterateConfigItems(this.databaseTables, this.config)
        
        console.log(`${Constants.ModTitle}: Weapons changes applied!`)
    }
}

File: Server/VAI-NRTP/Managers/WeaponsManager.cs
using SPTarkov.DI.Annotations;

namespace VAI.NRTP.Managers;

[Injectable(InjectionType.Singleton)]
public sealed class WeaponsManager : AbstractModManager
{
    protected override string ConfigName => "WeaponsConfig";

    protected override void AfterPostDb()
    {
        Helper.IterateConfigItems(DatabaseTables, Config);

        Constants.GetLogger().Info($"{Constants.ModTitle}: Weapons changes applied!");
    }
}
